tests/test_SwapsExternal.py

```diff
@@ -0,0 +1,54 @@
+import pytest
+from brownie import Contract, Wei, reverts
+from fixedint import *
+import shared
+
+def test_swapExternal(accounts, sovryn, RBTC, SUSD, priceFeeds):
+    SUSD.approve(sovryn.address, 1e18) 
+
+    tx = sovryn.swapExternal(SUSD.address, RBTC.address, accounts[0], accounts[0], 1e18, 0, 0, b'', priceFeeds.address)
+    
+    external_swap = tx.events['ExternalSwap']
+    assert(external_swap['user'] == accounts[0])
+    assert(external_swap['sourceToken'] == SUSD.address)
+    assert(external_swap['destToken'] == RBTC.address)
+    assert(external_swap['sourceAmount'] == 1e18)
+    assert(external_swap['sourceAmount'] == tx.return_value[1])
+    assert(external_swap['destAmount'] == tx.return_value[0])
```

Comment:
better to also check the actual value. it should be the same as the expected one.